network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -7,6 +7,7 @@
 
 {% block og-title %}
   <meta property="og:title" content="privacy not included - {{ product.name }}" />
+  <link rel="canonical" href="{{ request.scheme }}://{{ request.get_host }}{{ request.get_full_path }}" />
```

Comment:
Does this require canonical links for the homepage, too?

Reply:
I added this here because it's something Talk asks for specifically, and it's only on product pages.

Reply:
The block name doesn't quite make sense. I'm going to update that